pkg/imgutil/imgutil.go

```diff
@@ -200,6 +201,13 @@ func IsErrHTTPResponseToHTTPSClient(err error) bool {
 	return strings.Contains(err.Error(), unexposed)
 }
 
+// IsErrConnectionRefused return whether err is
+// "connect: connection refused"
+func IsErrConnectionRefused(err error) bool {
+	const errMessage = "connect: connection refused"
+	return strings.Contains(err.Error(), errMessage)
```

Comment:
Can't we use `errors.Is`?

Reply:
I think I can make a single PR to make all error message checks like `errors.Is`